packages/back-end/src/models/ProjectModel.ts

```diff
@@ -122,4 +118,69 @@ export class ProjectModel extends BaseClass {
       },
     };
   }
+
+  protected async afterCreate(
+    doc: ProjectInterface,
+    writeOptions?: never,
+  ): Promise<void> {
+    await super.afterCreate(doc, writeOptions);
+    // Refresh SDK payload cache for all environments
+    const payloadKeys = getPayloadKeysForAllEnvs(
+      this.context as ReqContext | ApiReqContext,
+      [doc.id],
+    );
+    refreshSDKPayloadCache(
+      this.context as ReqContext | ApiReqContext,
+      payloadKeys,
+    ).catch((e) => {
+      logger.error(
+        e,
+        "Error refreshing SDK payload cache after project create",
+      );
+    });
+  }
```

Comment:
Do we need this `afterCreate` hook?  When a project is first created, it has no features or experiments.  It's not possible for it to affect any SDK connections yet.